package.json

```diff
@@ -30,12 +30,14 @@
         "@testing-library/jest-dom": "^5.16.4",
         "@testing-library/react": "^12.1.4",
         "cypress": "^13.10.0",
+        "cypress-multi-reporters": "^1.6.4",
         "cypress-tags": "^1.2.2",
         "enzyme": "^3.11.0",
         "enzyme-adapter-react-16": "^1.15.8",
         "eslint-plugin-cypress": "^2.12.1",
         "identity-obj-proxy": "^3.0.0",
         "jest-enzyme": "^7.1.2",
+        "mochawesome": "^7.1.3",
```

Comment:
Just out of interest, why did we need this additional requirement? Is it required by report portal?

Reply:
Good catch! I think this is not needed... let me remove it :)